src/test/regress/expected/isolation_ensure_dependency_activate_node.out

```diff
@@ -1446,6 +1725,389 @@ count
 1              
 run_command_on_workers
 
+(localhost,57637,t,1)
+(localhost,57638,t,1)
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+master_remove_node
+
+               
+               
+
+starting permutation: s1-print-distributed-objects s1-begin s1-add-worker s2-public-schema s2-create-distributed-function s1-commit s2-print-distributed-objects
+?column?       
+
+1              
+step s1-print-distributed-objects: 
+    SELECT 1 FROM master_add_node('localhost', 57638);
+
+    -- print an overview of all distributed objects
+    SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object ORDER BY 1;
+
+    -- print if the schema has been created
+    SELECT count(*) FROM pg_namespace where nspname = 'myschema';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_namespace where nspname = 'myschema';$$);
+
+    -- print if the type has been created
+	SELECT count(*) FROM pg_type where typname = 'tt1';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_type where typname = 'tt1';$$);
+
+    -- print if the function has been created
+    SELECT count(*) FROM pg_proc WHERE proname='add_tt1';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_proc WHERE proname='add_tt1';$$);
+
+    SELECT master_remove_node('localhost', 57638);
+
+?column?       
+
+1              
+pg_identify_object_as_address
+
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+master_remove_node
+
+               
+step s1-begin: 
+    BEGIN;
+
+step s1-add-worker: 
+	SELECT 1 FROM master_add_node('localhost', 57638);
+
+?column?       
+
+1              
+step s2-public-schema: 
+    SET search_path TO public;
+
+step s2-create-distributed-function: 
+	CREATE TYPE tt1 AS (a int, b int);
```

Comment:
Why do we have this type in the test? We can have it, but I what I want to learn is can we still have `<waiting ...>` even if there is no  type? If not, we should probably be able  to see it, otherwise,  we could have nodes without the function via  concurrent master_add_node and create_distributed_function

Reply:
If I remove `CREATE TYPE ..` statement from `s2-create-distributed-function` step, and later use `s2-create-type` before `s2-create-distributed-function`, we indefinitely wait. I guess the type creation waits until s1 transaction is committed to move forward.

Should I create a new permutation for creating a distributed function that does not depend on a custom type? Does that sound good?

Reply:
Why not we remove `s2-create-type` at all for the permutations that are related to functions? We've some other tests which follows the dependency graph, ad this point we're mostly targeting to show that function creation works fine with add_node?

Reply:
We discussed it online. I changed the test steps to distribute simple functions without any custom type dependencies